source/runbooks/dr-process.html.md.erb

```diff
@@ -0,0 +1,362 @@
+
+
+### Will it be returning
+
+
+If this is a test account, waiting may be the easier idea. If AWS can promise us this is a temporary issue with an estimated fix time, we can pass this to the team and agree our next step.
+
+
+### Rebuild account
+
+
+Assuming this is a production account with no time given to return, please follow the steps.
+
+
+#### Workflows
+
+
+Rather than rebuilding an account through workflows, I would run steps manually to easier debug issues, however, technically, you could start to run the account creation process with our github workflows.
+
+**Member Accounts** - Located [here](https://github.com/ministryofjustice/modernisation-platform/actions/workflows/new-environment.yml), running the new environment workflow should begin the steps of an account creation for member accounts.
+
+**Core Accounts** - Located [here](https://github.com/ministryofjustice/modernisation-platform/actions), on the left you will see the deployment workflows for all core based accounts, required for the building on the platform.
+
+However, I would run manually just to monitor the plan, help troubleshoot issues, as sometimes these scenarios are not as cut and dry as all infrastructure has vanished from one account.
+
+
+#### Manually
+
+
+1.) Run a terraform plan against the production workspace, in modernisation-platform .
+
+2.) All changes should appear, using your admin credentials, run an apply.
+
+3.) Assuming the apply runs as planned, login to the account and check that vpcs, etc look correct.
+
+4.) In modernisation-platform-environments , run terraform against the correct account.
+
+
+
+### Things to remember
+
+
+- Statefile for the lost account will exist, this might need to be removed.
+- Any backups on the account or volumes will be lost.
+
+
+## Disaster Recovery Running outside of London.
+
+
+If the London infrastructure is to be down for the foreseeable future, running from another region is the next step.
+
+
+
+### Informing teams
+
+
+Firstly, inform the following channels of the situation and put forward a contactable member of staff from our team.
+
+
+`#ask-modernisation-platform`
+
+
+`#modernisation-platform`
+
+
+`#modernisation-platform-alerts`
+
+
+Supply the ETAs, lead incident numbers, links to where updates will be posted etc. We will mention our commitment to regularly we will update the incident, however until we know what is expected this is an unknown.
+
+
+### Cross Region Infrastructure in the Platform
+
+
+| Technology | Information | Replicated? | Replication Region |
+|----------------- |------------------------------------------------ |------------- |-------------------- |
+| S3 | Contains the platforms terraform state files. | Yes | eu-west-1 |
+| KMS | Contains encryption keys for the platform. | No | - |
+| Secrets Manger | Contains platform secrets. | Yes | eu-west-1 |
+| IAM Policies | Across all regions. | - | - |
```

Comment:
what about IAM roles and users?

Reply:
Changed to be IAM in general, not just the policies.